src/auth/dto/create-user.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsString,
+  MinLength,
+  Matches,
+  IsNotEmpty,
+  IsEmail,
+  Length,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+export class CreateUserDto {
+  @IsString()
+  @ApiProperty({
+    example: 'username',
```

Comment:
PullProofAI

Consider adding a more descriptive error message for the `name` field validation.
```diff
-   @IsNotEmpty({ message: 'First name is required' })
+   @IsNotEmpty({ message: 'Name is required' })
```



<!-- This is an auto-generated comment by PullProofAI -->